signature/policy_reference_match_test.go

```diff
@@ -57,7 +57,7 @@ func TestParseImageAndDockerReference(t *testing.T) {
 type refImageMock struct{ reference.Named }
 
 func (ref refImageMock) Reference() types.ImageReference {
-	return refImageReferenceMock{ref.Named}
+	return refImageReferenceMock(ref)
```

Comment:
I don't see golangci complaining about it. Why is that needed?

Reply:
S1016: should convert ref (type refImageMock) to refImageReferenceMock instead of using struct literal